lib/vanagon/platform/rpm.rb

```diff
@@ -45,15 +45,15 @@ def package_name(project)
         "#{project.name}-#{project.version}-#{project.release}.#{project.noarch ? 'noarch' : @architecture}.rpm"
       end
 
-      def output_dir(target_repo = "products")
-        super
-      end
-
       # Method to derive the directory for source artifacts
       #
       # @param target_repo [String] repo the source artifacts are targeting
-      def source_output_dir(target_repo = "products")
-        @source_output_dir ||= File.join(@os_name, @os_version, target_repo, 'SRPMS')
+      def source_output_dir(target_repo = nil)
+        if target_repo.nil?
+          super
```

Comment:
I figured I'd default to the platform tag with the builder architecture, but I can change this to default to the srpm/source directories like we do for the enterprise repos. That's quite logical to do